### PR TITLE
Fixed #182:Hide appbar in landscape orientation

### LINKED ISSUE #182: Hide appbar in landscape orientation
On mobile devices, need to hide the app bar layout as it takes up major space.

Further improvements ->

calculate whether to show app bar or not depending on available space(eg. tablets have more space)

### PR DESCRIPTION
**Pull request by aayushsingla**

### CODE REVIEW

File: app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java [hunk 1]
```diff
@@ -42,12 +46,14 @@
...
      */
-    public FaqViewHolder(@NonNull final View itemView, SparseBooleanArray sba) {
+    public FaqViewHolder(@NonNull final View itemView, SparseBooleanArray sba, Context mContext) {
         super(itemView);
         ButterKnife.bind(this, itemView);
         this.sba = sba;
         sba.put(getAdapterPosition(), false);
-        itemView.setOnClickListener(v -> showAnswer());
+        setCopyListener(questionTV,mContext);
+        setCopyListener(answerTV,mContext);
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java]: no need to pass activity context all the way here. We can do `itemView.getContext()`

File: app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java [hunk 2]
```diff
@@ -90,4 +96,14 @@ public void setAnswerTV(@NonNull final String answer) {
...
 
     }
+
+   private void setCopyListener(TextView textView,Context context){
+       textView.setOnLongClickListener(v -> {
+           ClipboardManager clipboard = (ClipboardManager) context.getSystemService(Context.CLIPBOARD_SERVICE);
+           ClipData clip = ClipData.newPlainText(textView.toString(),textView.getText()+"");
+           clipboard.setPrimaryClip(clip);
+           Toast.makeText(context,"Copied To Clipboard",Toast.LENGTH_LONG).show();
+           return false;
```
Comment (Rushi98) [inline app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java]: ~`return false`~ `return true` we are consuming the long click

File: app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java (hunk 1 above)
Comment (Rushi98) [reply app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java]: I think it's hard to long press questionTv avoiding answerTv (and vice versa). Better put single copyListener and copy both question and answer  - maybe something like
```
[Q] What is the question?
[A] Anyway, this is answer.
```

**Comment by Rushi98** [comment]
See the sugggestions. Otherwise, code works.

File: app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java (hunk 1 above)
Comment (aayushsingla) [reply app/src/main/java/com/macbitsgoa/ard/viewholders/FaqViewHolder.java]: Ok, Will copy both question and answer together.

**Comment by kukreja-vikramaditya** [comment]
@Rushi98 Is DetailFragment correct name? or you have something else in mind? Like AboutFragment or similar?

**Pull request closed without merging**